backend/coreapp/asm_diff_wrapper.py

```diff
@@ -93,12 +100,9 @@ def run_objdump(target_data: bytes, platform: str, config: asm_differ.Config, la
                         },
                     )
                 except subprocess.CalledProcessError as e:
-                    logger.error(e)
-                    logger.error(e.stderr)
-                    return None
+                    raise ObjdumpError(f"Error running objdump: {str(e)}")
```

Comment:
Does str(e) include the e.stdout + e.stderr? Wondering if we'd lose any messages that would help identify any errors?

Reply:
I'm not sure...good point